Cargo.lock

```diff
@@ -77,19 +77,19 @@ version = "0.7.7"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "5a824f2aa7e75a0c98c5a504fceb80649e9c35265d44525b5f94de4771a395cd"
 dependencies = [
- "getrandom 0.2.11",
```

Comment:
@enthusiastmartin did you run `cargo update`? 

Was this necessary for something, shall we keep the version changes or revert?

Reply:
restored old Cargo.lock to fixb uild